frontend/components/BurgerMenu/index.jsx

```diff
@@ -0,0 +1,36 @@
+import LinksLanguages from "../LinksLanguages";
+import LinksMenu from "../LinksMenu";
+import styles from "./burgerMenu.module.css";
+
+export default function BurgerMenu({
+  isOpenedMainHeader,
+  openBurgerMenuAndCloseMainHead,
+  setIsOpened,
+  setIsOpenedMainHeader,
+  categories,
+  isOpened,
+}) {
+  return (
+    <nav className={`${isOpenedMainHeader && styles.top_nav_bar_block_burger}`}>
+      <div
+        className={styles.wrapper_burger}
+        onClick={() =>
+          openBurgerMenuAndCloseMainHead({
+            event,
+            setIsOpened,
+            setIsOpenedMainHeader,
+          })
+        }
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/event
This approach to working with an event seems outdated. Please use the event that was passed to the callback as an argument:
```suggestion
        onClick={(event) =>
          openBurgerMenuAndCloseMainHead({
            event,
            setIsOpened,
            setIsOpenedMainHeader,
          })
        }
```

Reply:
And from PHPStorm IDE (Jetbrains):
<img width="665" alt="Screenshot 2021-10-04 at 18 05 18" src="https://user-images.githubusercontent.com/13319258/135875847-2d78c987-0b82-44e4-ada4-781213c7db5d.png">

Reply:
Fixed in the Fix event commit